docs/tutorial_advanced.rst

```diff
@@ -715,13 +715,19 @@ Besides the stage directory, additional mount points can be specified through th
                                             ('/path/to/host/dir2', '/path/to/container/mount_point2')]
 
 
-The container filesystem is ephemeral, therefore, ReFrame mounts the stage directory under ``/rfm_workdir`` inside the container where the user can copy directories/files as needed.
-The aforementioned directories/files will then be available inside the stage directory after the container execution finishes.
-This is very useful if the above directories/files are going to be used for the sanity/performance checks.
-If this is the case, the user should overwrite the default command executed by the container, using the :attr:`command <reframe.core.containers.ContainerPlatform.command>` to include the appropriate copy commands.
-In this test, we have changed the directory that the container is going to run the commands using the `--pwd=/rfm_workdir` through the :attr:`options <reframe.core.containers.ContainerPlatform.options>`.
-Since this is the directory that ReFrame mounts the stage directory inside the container, the files created there are available after the container execution finishes.
-Therefore, the ``release.txt`` where the output of the ``cat /etc/os-release`` is redirected, can be used in the subsequent sanity function:
+The container filesystem is ephemeral, therefore, ReFrame mounts the stage directory under ``/rfm_workdir`` inside the container where the user can copy artifacts as needed.
+The aforementioned artifacts will then be available inside the stage directory after the container execution finishes.
+This is very useful if the above artifacts are going to be used for the sanity/performance checks.
+If the copy is not performed by the default container commands, the user should overwrite the default command executed by the container, using the :attr:`command <reframe.core.containers.ContainerPlatform.command>` to include the appropriate copy commands.
+In the current test, the output of the ``cat /etc/os-release`` is made available outside the container since we have used the command:
+
+.. code-block:: bash
+
+    bash -c 'cat /etc/os-release | tee /rfm_workdir/release.txt'
```

Comment:
You should mention perhaps that you also print in stdout to demonstrate also how the stdout of the container can be used in sanity and performance checking as usual.